### PR TITLE
Initial anacondarecipes build

### PR DESCRIPTION
**Pull request by mcg1969**
actions:
- added license_family, description, doc_url, doc_source_url, dev_url
- moved sha256 to header

### CODE REVIEW

**Comment by mcg1969** [comment]
linux-ppc64le fails with:
```
-- The CXX compiler identification is GNU 11.2.0                                                                         
-- Detecting CXX compiler ABI info                                                                                       
-- Detecting CXX compiler ABI info - done                                                                                
-- Check for working CXX compiler: $BUILD_PREFIX/bin/powerpc64le-conda-linux-gnu-c++ - skipped                           
-- Detecting CXX compile features                                                                                        
-- Detecting CXX compile features - done                                                                                 
-- Fetching BoostFetch.cmake                                                                                             
-- Fetching boostorg/cmake:develop                                                                                       
-- Configuring incomplete, errors occurred!                                                                              
```
s390x fails with:
```
CMake Error at                                                                                                           
/opt/conda/conda-bld/boost_mp11_1657824180404/_build_env/share/cmake-3.22/Modules/ExternalProject.cmake:2666 (message):  
  error: could not find git for clone of boostorg_cmake-populate                                                         
```

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+aggregate_check: false
+channels:
+  - sfe1ed40
```
Comment (katietz) [inline abs.yaml]: same thing as for libdate 3.0.1 PR

**Review by katietz — CHANGES_REQUESTED** [review]

File: abs.yaml (hunk 1 above)
Comment (mcg1969) [reply abs.yaml]: @katietz thanks for the comment. Note that I do not intend to keep the abs.yaml file in the final build. I am probably doing the initial build wrong here, but I tend to develop in the services channel initially to ensure the build is correct. Once that passes, I remove abs.yaml, add it to the aggregate, and so forth. I apologize if I'm not doing things in the right order.

Comment (mcg1969) [reply abs.yaml]: removed abs.yaml, added to aggregate

**Comment by mcg1969** [comment]
I need to open a new pull request to get around prefect caching  issues

**Pull request closed without merging**